packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -313,6 +302,27 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
     }
   }
 
+  // If it's a calendar remove it from the SelectedCalendars
+  if (credential.app?.categories.includes(AppCategories.calendar)) {
+    const calendar = await getCalendar(credential);
+
+    const calendars = await calendar?.listCalendars();
```

Comment:
I like this approach for our current selected calendars, but I think we can improve this. I'm going to approve for now as this is a great solution.